package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "fastify-compress",
+  "version": "0.1.0",
+  "description": "Fastify compression utils",
+  "main": "index.js",
+  "dependencies": {
+    "fastify-plugin": "^0.1.1",
+    "iltorb": "^2.0.2",
```

Comment:
this is a native dependency. I would not add it as a dependency here, bur rather treat it as optional (in case it fails to install).

Reply:
👍